### ISSUE TITLE
LTP testcase chdir02 fails to chdir to "/" or "/tmp"

### ISSUE THREAD

**Issue by arjun4084346** [oscarlab/graphene#25]
Failed LTP test : chdir02
I see that, path_lookupat() in shim_namei.c is not setting the DENTRY_ISDIRECTORY bit in dent->state, returning -ENOTDIR at
https://github.com/oscarlab/graphene/blob/master/LibOS/shim/src/sys/shim_getcwd.c#L82

**Comment by omeg** [comment]
@mkow 
This testcase seems to work as of commit 4fb1278.

Repro steps:
1. `cd LibOS/shim/test/apps/ltp`
2. `make`
3. `make regression`
4. Check `passed.csv`

Running a single case as described in README doesn't seem to work:
```
$ ~/graphene/Runtime/pal-Linux opt/ltp/testcases/bin/chdir02
Can't find any manifest, will run without one.
<hangs>
```

**Comment by mkow** [comment]
cd-ing to test's directory fixed the problem.
I ran this test ~1000 times on current master and it seems to work.

**Comment by mkow** [comment]
I fixed the readme in #458.

**Comment by mkow** [comment]
The fix was merged, closing.
